.github/workflows/ci.yaml

```diff
@@ -12,20 +12,20 @@ jobs:
   test:
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
```

Comment:
new poetry (v8) doesn't work with 3.7. That's why I removed it. Other components are already on 3.10. Reason I did not add 3.10 is because Rucio DID finder does not work with 3.10 (they are 3.8).